opentelekomcloud/acceptance/rds/data_source_opentelekomcloud_rds_instances_v3_test.go

```diff
@@ -0,0 +1,83 @@
+package acceptance
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/common"
+)
+
+func TestAccRdsInstanceDataSource_basic(t *testing.T) {
+	dataSourceName := "data.opentelekomcloud_rds_instance_v3.test"
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckRdsInstanceV3Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRdsInstanceDataSource_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(dataSourceName, "instances.#", regexp.MustCompile(`\d+`)),
+					resource.TestCheckResourceAttrSet(dataSourceName, "instances.0.name"),
+				),
+			},
+		},
+	})
+}
+
+func testAccRdsInstanceDataSource_basic(rName string) string {
+	return fmt.Sprintf(`
+resource "opentelekomcloud_vpc_v1" "test" {
+  name = "%s"
+  cidr = "192.168.0.0/16"
+}
+
+resource "opentelekomcloud_vpc_subnet_v1" "test" {
+  name       = "%s"
+  cidr       = "192.168.0.0/24"
+  gateway_ip = "192.168.0.1"
+  vpc_id     = opentelekomcloud_vpc_v1.test.id
+}
+
+resource "opentelekomcloud_networking_secgroup_v2" "test" {
+  name = "%s"
+}
```

Comment:
We don't need to create network infra, can you use predefined infra like in `rds_instance_v3` test?

- `common.DataSourceSecGroupDefault`
- `common.DataSourceSubnet`